umami.theme

```diff
@@ -34,4 +49,4 @@ function umami_form_alter(&$form, FormStateInterface $form_state, $form_id) {
   if ($form['#id'] == 'search-block-form') {
     $form['actions']['submit']['#attributes']['class'][] = 'visually-hidden';
   }
-}
+}
```

Comment:
There should be a new line at the end of the file.

Reply:
Adding the new line in